examples/slurm-mpi-hello/README.md

```diff
@@ -0,0 +1,79 @@
+"Hello, World" with MPI on a Slurm cluster
+==========================================
+
+Slurm is an open-source job scheduling system for Linux clusters, most frequently used for high-performance computing (HPC) applications.
+Many HPC applications are built on the Message Passing Interface (MPI) standard, which allows multiple processes in an application to communicate across nodes.
+
+This example demonstrates how to build and run simple "hello, world" MPI application on a Slurm cluster.
+It assumes that you have already deployed a cluster with a login node and at least one compute node, and that Slurm has already been set up and configured.
+It also assumes the presence of a shared NFS filesystem on all nodes in the cluster, which is given the path `/shared` in this example.
+If you haven't configured a Slurm cluster yet, see the [Slurm guide](/docs/slurm-cluster.md) for information on building a GPU-enabled Slurm cluster.
+
+1. **Install the OpenMPI packages:**
+    On many clusters, MPI libraries and tools are built from source to take advantage of the specific cluster hardware.
+    In this example, we'll just use the OpenMPI provided by Ubuntu to demonstrate running an MPI job.
+    To install OpenMPI, run the following `bootstrap-mpi.yml` script using Ansible.
+    If you only want to install on a subset of nodes, use the `-l ${HOST_GROUP}` argument to restrict where this playbook is run.
+    ```
+    $ ansible-playbook -i ${INVENTORY_FILE} [-l ${HOST_GROUP}] bootstrap-mpi.yml
```

Comment:
I was referring to these paths, i.e. `bootstrap-mpi.yml` here becomes `examples/slurm-mpi-hello/bootstrap-mpi.yml`